src/axom/primal/tests/primal_in_sphere.cpp

```diff
@@ -6,104 +6,188 @@
   }
 
   // Test some points that are on the circumsphere
   {
-    PointType q1 = PointType::make_point(1, 1, 1);
+    PointType q1 {1, 1, 1};
     EXPECT_FALSE(in_sphere(q1, p0, p1, p2, p3));
+    EXPECT_FALSE(in_sphere(q1, tet));
 
-    PointType q2 = PointType::make_point(-1, 1, 1);
+    PointType q2 {-1, 1, 1};
     EXPECT_FALSE(in_sphere(q2, p0, p1, p2, p3));
+    EXPECT_FALSE(in_sphere(q2, tet));
   }
 
   // Test some points that are outside the circumsphere
   {
-    PointType q1 = PointType::make_point(1.1, 1, 1);
+    PointType q1 {1.1, 1, 1};
     EXPECT_FALSE(in_sphere(q1, p0, p1, p2, p3));
+    EXPECT_FALSE(in_sphere(q1, tet));
 
-    PointType q2 = PointType::make_point(-1.1, 1, 1);
+    PointType q2 {-1.1, 1, 1};
     EXPECT_FALSE(in_sphere(q2, p0, p1, p2, p3));
+    EXPECT_FALSE(in_sphere(q2, tet));
+  }
+}
+
+TEST(primal_in_sphere, compare_to_sphere_orientation)
+{
+  const int DIM = 3;
+  using PointType = primal::Point<double, DIM>;
+  using TetrahedronType = primal::Tetrahedron<double, DIM>;
+  using SphereType = TetrahedronType::SphereType;
+
+  std::vector<PointType> queryPoints = {PointType {2, 0, 0},
+                                        PointType {0.359547, 0.731502, 0.370846}};
+
+  // define some constants for regular tetrahedron
+  const double c[3] = {std::sqrt(2) / 3, std::sqrt(6) / 3, 1. / 3};
+  std::vector<TetrahedronType> tets = {
+    // A regular tetrahedron centered at the origin
+    TetrahedronType {PointType {0, 0, 1},
+                     PointType {2 * c[0], 0, -c[2]},
+                     PointType {-c[0], -c[1], -c[2]},
+                     PointType {-c[0], c[1], -c[2]}},
+
+    TetrahedronType {PointType {0.942188, 0.211634, 0.395159},
+                     PointType {0.9105, 0.114406, 0.83865},
+                     PointType {0.855718, 0.660449, 0.911065},
+                     PointType {0.393771, 0.572654, 0.471824}}};
+
+  std::vector<primal::OrientationResult> orientations = {
+    primal::ON_POSITIVE_SIDE,
+    primal::ON_POSITIVE_SIDE,
+  };
+
+  const int numChecks = queryPoints.size();
+  for(int i = 0; i < numChecks; ++i)
+  {
+    const auto& query = queryPoints[i];
+    const auto& tet = tets[i];
+    const auto& exp_orient = orientations[i];
+    const SphereType sphere = tet.circumsphere();
+
+    SLIC_INFO(
+      axom::fmt::format("Testing point {} against circumsphere {} "
+                        "\n\t for tet {}  with signed volume {}"
+                        "\n\t signed distance to sphere boundary is {}",
+                        query,
+                        sphere,
+                        tet,
+                        tet.signedVolume(),
+                        sphere.computeSignedDistance(query)));
+
+    // check that each vertex is on the sphere boundary
+    for(int i = 0; i <= DIM; ++i)
+    {
+      EXPECT_TRUE(sphere.getOrientation(tet[i]) == primal::ON_BOUNDARY);
+    }
+
+    // check that the orientation matches expectations
```

Comment:
These tests ensure that we get the same result for `in_sphere()` as we do when we compute the circumsphere of a triangle/tet and then call `getOrientation()`